pkgs/applications/video/manim/default.nix

```diff
@@ -64,8 +64,10 @@ in python.pkgs.buildPythonApplication rec {
 
   pythonRelaxDeps = [
```

Comment:
The version locks on these dependencies seem to [already](https://github.com/ManimCommunity/manim/commit/61a2d05b69862a49692bef01e776ff305beadf39)  have been relaxed on the upstream branch ([watchdog](https://github.com/ManimCommunity/manim/blob/main/pyproject.toml#L57) and [isosurfaces](https://github.com/ManimCommunity/manim/blob/main/pyproject.toml#L36)). We should probably have a comment here, noting to change that, when a new release is made.